apps/ui-kit/src/lib/components/atoms/radio-button/RadioButton.tsx

```diff
@@ -53,7 +53,7 @@ function RadioButton({
                 />
                 <span
                     className="absolute
-                    text-neutral-40 peer-checked:text-primary-30 peer-disabled:opacity-40 peer-checked:peer-disabled:text-neutral-40 dark:text-neutral-60 dark:peer-checked:peer-disabled:text-neutral-40 [&_svg]:h-6 [&_svg]:w-6
+                    text-neutral-40 peer-checked:text-primary-30 peer-disabled:opacity-40 peer-checked:peer-disabled:text-neutral-40 dark:text-neutral-60 dark:peer-checked:text-primary-80 dark:peer-checked:peer-disabled:text-neutral-40 [&_svg]:h-6 [&_svg]:w-6
```

Comment:
It doesn't look like the same color on the radio button

![Captura desde 2024-11-25 09-01-33](https://github.com/user-attachments/assets/78ba3059-c8f7-41c6-b027-677d2be32161)

![image (21)](https://github.com/user-attachments/assets/0b559b73-8173-4de0-a94b-98e0773b3b64)